sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1063,15 +1060,14 @@ static ipv6_addr_t *_src_addr_selection(gnrc_netif_t *netif,
             return ptr;
         }
         /* Rule 2: Prefer appropriate scope. */
-        /* both link local */
         uint8_t candidate_scope = _get_scope(ptr);
         if (candidate_scope == dst_scope) {
             DEBUG("winner for rule 2 (same scope) found\n");
-            winner_set[i] += RULE_2A_PTS;
+            winner_set[i] += RULE_2_PTS;
         }
         else if (candidate_scope < dst_scope) {
```

Comment:
`If Scope(SA) < Scope(SB): If Scope(SA) < Scope(D), then prefer SB and`

To me this means prefer scope closed to D but that are greater than D, in that case you should get points if you are bugger than destination

```suggestion
        else if (candidate_scope > dst_scope) {
```

Reply:
I think you are right on that.